http/client_test.go

```diff
@@ -244,6 +244,78 @@ func (s *HTTPClientTestSuite) TestResourceLimitExceeded(c *gc.C) {
 	c.Assert(err, gc.ErrorMatches, `Resource limit exeeded at URL http://.*`)
 }
 
+func (s *HTTPClientTestSuite) TestRetryHttpDate(c *gc.C) {
+	count := 0
+	var t0 time.Time
+	srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		data, _ := ioutil.ReadAll(req.Body)
+		c.Check(string(data), gc.Equals, "request body")
+		count++
+		switch count {
+		case 1:
+			t0 = time.Now()
+			t0 = t0.Add(time.Duration(10) * time.Second)
```

Comment:
Since this test suit is not using test clocks, adding 10s here will make the test suite take a while. Can we make it 1s like in the other test.

Reply:
done